Lib/ntpath.py

```diff
@@ -66,16 +76,6 @@ def normcase(s):
             return _LCMapStringEx(_LOCALE_NAME_INVARIANT,
                                   _LCMAP_LOWERCASE,
                                   s.replace('/', '\\'))
-except ImportError:
```

Comment:
Why has this function been moved? Does this speed things up?

Reply:
_Unnesting_, but as you requested I'll revert it.

Reply:
Does "unnesting" speed anything up, or have any other impact that would be visible to users of the code? If not, then it's a cosmetic change, and doesn't belong as part of this PR :-)

Reply:
No, it doesn't speed anything up, it only improves readability. I'll revert it in the next commit.